cmake/toolchains/mt.runtime.win32.toolchain.cmake

```
@@ -27,6 +27,11 @@ if(use_static_runtime)
         foreach(build_type "" "_DEBUG" "_MINSIZEREL" "_RELEASE" "_RELWITHDEBINFO")
             set(flag_var "CMAKE_${lang}_FLAGS${build_type}_INIT")
             string(REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
+            if (build_type STREQUAL "_DEBUG")
```

```suggestion
            if (${flag_var} MATCHES "_DEBUG")
```
is it more correct?
as `build_type` is defined after cmake is configured for Ninja Multi-Config / MSVC generators

---

I don't think so. Because we are using the build_type from `foreach(build_type "" "_DEBUG" "_MINSIZEREL" "_RELEASE" "_RELWITHDEBINFO")`, which is already defined? @ilya-lavrenov

---

ok, I see.
I mixed it with `CMAKE_BUILD_TYPE`